operations/commit_queue.go

```diff
@@ -0,0 +1,93 @@
+package operations
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/evergreen-ci/evergreen/rest/client"
+	restModel "github.com/evergreen-ci/evergreen/rest/model"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+func CommitQueue() cli.Command {
+	const (
+		listFlagName   = "list"
+		deleteFlagName = "delete"
+		itemFlagName   = "item"
+	)
+
+	return cli.Command{
+		Name:  "commit-queue",
```

Comment:
does it make sense to put this under admin rather than a top level command?

Reply:
we discussed this in person and decided to leave it as is since:

1. a regular user will interact with it to delete their own items from the queue
2. The commit queue didn't seem to fit with the rest of what's under admin now.